### ISSUE TITLE
add `lint:fix` to package.json

### ISSUE THREAD

**Issue by brian-smith-tcril** [openedx/paragon#2514]
We have this in `frontend-template-application` https://github.com/openedx/frontend-template-application/blob/984c19a3a617c2686fbdb412c6eec637d3725ccf/package.json#L16.

It'd be nice to be able to tell people who have lint errors on their PR to try running `npm run lint:fix` to quickly resolve lint issues.

**Comment by adamstankiewicz** [comment]
Good idea to have! Also, perhaps as a separate task, we may want to have improved documentation (in the README?) around what NPM scripts exist for this repo (that's a lot now 😄).

**Comment by adamstankiewicz** [comment]
@brian-smith-tcril Secondary thought, are you aware of any tooling in place to ensure common NPM scripts exist on repos? Similar to how PRs automatically open to add missing standard GitHub action workflows, etc.

Maybe thats a level of standardization too far, but I could see some script automatically running to open a PR to add a `lint:fix` script to JS repos that don't have it. Beyond this, or now that we're introducing a Paragon CLI to more easily work with custom `@edx/brand` packages during local development, it'd be great to more automatically add/maintain the following NPM scripts to save needing to manually go through each repo ourselves:

```json
{
  "scripts": {
    "start:with-theme": "paragon install-theme && npm start",
    "start:with-theme": "paragon install-theme && npm run build",
  }
}
```

Food for thought. Not sure if the juice is worth the squeeze to build out such automated tooling for modifying repo's common configured NPM scripts.

**Comment by brian-smith-tcril** [comment]
> are you aware of any tooling in place to ensure common NPM scripts exist on repos?

not off the top of my head, it does seem like something we could include in a github action similar to the one we discussed adding for dependency checks

**Comment by brian-smith-tcril** [comment]
> we may want to have improved documentation (in the README?) around what NPM scripts exist for this repo (that's a lot now ).

good call https://github.com/openedx/paragon/issues/2515

**Comment by PKulkoRaccoonGang** [comment]
This is a great idea, thanks Brian! I admit, I manually add parameters for fixing before I need to run the linter :)

As an additional point, should we add a separate command to lint the `www` directory? I periodically remove www from [.eslintignore](https://github.com/openedx/paragon/blob/master/.eslintignore#L4) in order to check the code and fix problems.

**Comment by peterkulko** [comment]
[inform]: We already have a partial result in the form of https://github.com/openedx/paragon/blob/release-23.x/package.json#L35

**Comment by brian-smith-tcril** [comment]
I'm going to close this as `lint:fix` was added in https://github.com/openedx/paragon/pull/3330.

We have #2515 to track documentation, and we can make new issues to track other npm script stuff we want to handle.
